src/Map/doc/index.rst

```diff
@@ -128,6 +132,39 @@ You can add markers to a map using the ``addMarker()`` method::
         ))
     ;
 
+Add Marker icons
+~~~~~~~~~~~~~~~~
+
+It is possible to add icon to Marker::
+
+        new Marker(
+            position: new Point(45.7640, 4.8357),
+            title: 'Lyon',
+            infoWindow: new InfoWindow(
+                headerContent: '<b>Lyon</b>',
+                content: 'The French town in the historic Rhône-Alpes region, located at the junction of the Rhône and Saône rivers.'
+            ),
+            icon: Icon::fromInlineSvg(
+                svg: '<svg>....</svg>'
+            )
+        ))
+
+You can add image icon (type 'url')::
+
+    Icon::fromUrl(
+        url: 'https://my-image.png',
+        width: 24, // default
+        height: 24, // default
+    )
+
+or SVG icon (type 'html')::
+
+    Icon::fromInlineSvg(
+        svg: '<svg>...</svg>',
+        width: 36,
+        height: 36, 
+    )
```

Comment:
Let's re-write this part:

```suggestion
.. versionadded:: 2.24

    ``Marker`` icon customizatisation is available since UX Map 2.24.

When returning a Map, it's quite common to want to customize ``Marker`` icon. It is possible thanks to ``icon`` parameter and ``Icon`` class::

        // It can be a UX Icon (requires `symfony/ux-icons`)
        $icon = Icon::fromUxIcon('fa:map-marker');
        // Or an URL pointing to an image
        $icon = Icon::fromUrl('https://example.com/marker.png');
        // Or a plain SVG content
        $icon = Icon::fromInlineSVG('<svg>(...)</svg>');

        new Marker(
            // ...
            icon: $icon
        ))
```